TESTING.md

```diff
@@ -444,7 +452,7 @@ To open the context menu:
 1. Login to Cody Pro account
 2. Create new chat
 3. Default model for new Pro users is Claude 3 Sonnet
-   - Default model for existing Pro users is their previously selected model
+    - Default model for existing Pro users is their previously selected model
```

Comment:
Double-checked the rendering is still correct. This diff came from auto-formatting, it's fixing the indentation
 
![CleanShot 2024-04-18 at 14 01 15@2x](https://github.com/sourcegraph/jetbrains/assets/1408093/58c6f325-0ff7-48d8-9481-e683f3aca825)